### ISSUE TITLE
Shiki is downloading all languages despite the config

### ISSUE THREAD

**Issue by nicholasnet** [shikijs/shiki#512]
As per documentation you should be able to restrict the language that needs to be downloaded however it seems like it is not working. I see that it is downloading all files and causing unnecessary network usage. This is my code.

**OS**: Mac
**Env**: Browser using Firefox

```
import { getHighlighter, type Highlighter, setCDN } from 'shiki'

setCDN('https://cdn.jsdelivr.net/npm/shiki/')
getHighlighter({
    theme: 'css-variables',
    langs: ['js', 'java', 'kotlin', 'markdown']
}).then(highlighter => {
    console.log(highlighter.codeToHtml(`console.log('shiki');`, { lang: 'js' }))
})
```
Is there anyway to resolve this issue.

**Comment by souporserious** [comment]
The markdown language will require all code grammars since you can have code blocks which can be any language. This does feel like a bug though and it would be nice to configure this and only include necessary languages for markdown since you probably never want every single one.

**Comment by antfu** [comment]
It should be fixed with v1.0, #557
